src/main/java/com/redhat/cloud/notifications/templates/Advisor.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.redhat.cloud.notifications.models.EmailSubscriptionType;
+import io.quarkus.qute.CheckedTemplate;
+import io.quarkus.qute.TemplateInstance;
+
+// Name needs to be "Advisor" to read templates from resources/templates/Advisor
+public class Advisor extends AbstractEmailTemplate {
+
+    @Override
+    public TemplateInstance getTitle(String eventType, EmailSubscriptionType type) {
+        if (type == EmailSubscriptionType.INSTANT) {
+            return Templates.newRecommendationInstantEmailTitle();
+        }
+
+        return null;
```

Comment:
Is that OK to return `null`? I can see code that could throw a NPE but I guess the `Advisor.isSupported` method will be called first and this `null` value should never be returned.

Reply:
Yep, maybe I should replace it with an exception?

Reply:
Yes, I would prefer an exception with a message that clearly explains what's wrong rather than a potential NPE in the future.